.circleci/config.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - run:
           name: Bandit SAST tool
           command: |
-            bin/dc exec -T shipchain-common bandit -r src
+            bin/dc exec -T shipchain-common bandit -r src -s B311
```

Comment:
Let's not override bandit rules on the command line - if necessary, we can configure rules in a bandit config file: https://bandit.readthedocs.io/en/latest/config.html

But actually, in this case, I think the most appropriate thing to do would be to exclude the test utils from the bandit scan. This is a good security check to enforce on all other code, but I don't think it's harmful to run random() in a unit test. Check the docs for the bandit -x CLI option.

Reply:
Got it :+1: 
I finally opted out for the following command:
`bandit -r src -x src/shipchain_common/test_utils.py`
which will allow us to ship the scan for the relevant file.

Reply:
Nice :+1: I just realized I told you to use the -x flag which is great but I think it'd simplify the change if we made a `.bandit` file with the following rule:
```
[bandit]
exclude: src/shipchain_common/test_utils.py
```
https://github.com/PyCQA/bandit#configuration